cypress/e2e/patient_spec/patient_files.cy.ts

```diff
@@ -0,0 +1,227 @@
+import { PatientEncounter } from "@/pageObject/Patients/PatientEncounter";
+import { PatientFiles } from "@/pageObject/Patients/PatientFiles";
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+
+const facilityCreation = new FacilityCreation();
+const patientEncounter = new PatientEncounter();
+const patientFiles = new PatientFiles();
+
+describe("Patient Files", () => {
+  beforeEach(() => {
+    // cy.loginByApi("raj");
+    // cy.visit("/");
+    // facilityCreation.selectFacility("MEDICAL FACILITY");
+
+    cy.loginByApi("nurse");
+    cy.visit("/");
+    facilityCreation.selectFacility("GHC payyanur");
+
+    patientEncounter
+      .navigateToEncounters()
+      .openFirstEncounterDetails()
+      .clickPatientDetailsButton();
+    patientFiles.clickFilesTab();
+  });
+
+  const timestamp = new Date().getTime();
```

Comment:
```suggestion
  const timestamp = new Date().getTime();
```

keep const used inside a single test it() , inside that , rather than outside

Reply:
tests are not failing in CI/CD but not in local since i made changes in local backend to sort the files in latest first order.
https://github.com/ohcnetwork/care_fe/pull/10774#issuecomment-2707742787

Reply:
moved the variables which are not being used by more than one tests to inside their tests.